Sources/web3swift/Tokens/ST20/Web3+SecurityToken.swift

```diff
@@ -89,13 +83,16 @@ public class SecurityToken: ISecurityToken, ERC20BaseProperties {
         self.transaction = transaction
         self.transaction.to = address
         self.abi = abi
+        // Forced because this should fail if contract is wrongly configured
+        let contract = web3.contract(Web3.Utils.erc20ABI, at: address)!
+        self.basePropertiesProvder = ERCBasePropertiesProvider(contract: contract)
     }
 
     // Must be 18!
```

Comment:
I guess this comment is not really valid.

Reply:
Was wondering about this too but not part of the PR. If we know what's correct here, I can fix in a separate PR.

Reply:
Thanks for the great explanation!